pkg/command/install/install.go

```diff
@@ -0,0 +1,239 @@
+`
+)
+
+// InstallOptions contains the options to the installation command.
+type InstallOptions struct {
+	version string
+
+	genericclioptions.IOStreams
+}
+
+// NewInstallOptions returns a InstallOptions.
+func NewInstallOptions(streams genericclioptions.IOStreams) *InstallOptions {
+	return &InstallOptions{
+		version:   "latest",
+		IOStreams: streams,
+	}
+}
+
+// NewInstallCommand creates the installation command which can install the operator in the kubernetes cluster.
+func NewInstallCommand(ctx *cmdcontext.RWContext, streams genericclioptions.IOStreams) *cobra.Command {
+	o := NewInstallOptions(streams)
+
+	cmd := &cobra.Command{
+		Use:     "install",
+		Short:   "Install the risingwave operator in the cluster",
+		Long:    "Install the risingwave operator in the cluster",
+		Example: installExample,
+		Run: func(cmd *cobra.Command, args []string) {
+			util.CheckErr(o.Complete(ctx, cmd, args))
+			util.CheckErr(o.Run(ctx, cmd, args))
+		},
+	}
+
+	cmd.Flags().StringVarP(&o.version, "version", "v", o.version, "the version of risingwave operator to install.")
+
+	return cmd
+}
+
+func (o *InstallOptions) Complete(ctx *cmdcontext.RWContext, cmd *cobra.Command, args []string) error {
+	if len(o.version) == 0 {
+		o.version = "latest"
+	}
+
+	return nil
+}
+
+// Run will run the command as followed:
+// 1. check cert-manager
+// 2. install cert-manager or give the installation guide
+// 3. wait cert-manager ready
+// 4. install risingwave operator.
+func (o *InstallOptions) Run(ctx *cmdcontext.RWContext, cmd *cobra.Command, args []string) error {
+
+	exist, err := hasOperator(ctx)
+	if err != nil {
+		return err
+	}
+	if exist {
+		fmt.Fprintln(o.Out, "RisingWave Operator already exists")
+		return nil
+	}
+	fmt.Fprintln(o.Out, "RisingWave Operator not exists, need to install it!")
+
+	// check cert-manager
+	exist, err = hasCertManagerCR(ctx)
+	if err != nil {
+		return err
+	}
+	if !exist {
+		fmt.Fprintln(o.Out, "Install the cert-manager!")
+		err := installCertManager(ctx)
+		if err != nil {
+			return fmt.Errorf("install cert-manager failed, %w", err)
+		}
+		fmt.Fprintln(o.Out, "Wait the cert-manager ready!")
+		err = waitCertManager(ctx)
+		if err != nil {
+			return fmt.Errorf("wait cert-manager failed, %w", err)
+		}
+	}
+
+	fmt.Fprintln(o.Out, fmt.Sprintf("Install the %s! risingwave-operator", o.version))
+	err = installOperator(ctx, o.version)
+	if err != nil {
+		return fmt.Errorf("install risingwave failed, %w", err)
+	}
+
+	fmt.Fprintln(o.Out, "RisingWave Operator has been installed")
+
+	return nil
+}
+
+func waitCertManager(ctx *cmdcontext.RWContext) error {
+	err := wait.PollImmediate(time.Second, time.Minute*TimeOut, func() (bool, error) {
+		ready, inErr := checkCertManagerReady(ctx)
+		if inErr != nil {
+			return false, inErr
+		}
+		return ready, inErr
+	})
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func checkCertManagerReady(ctx *cmdcontext.RWContext) (bool, error) {
+	list := &corev1.PodList{}
```

Comment:
Not only we need to wait before cert-manager's Pod's running, but also have to wait until it's fully set up. Otherwise we'll get an error when installing our webhook configurations.
Check [e2e/e2e.sh:L95](https://github.com/singularity-data/risingwave-operator/blob/048e7b7a7f0ea90fbdb2cccfee3146182674089c/e2e/e2e.sh#L95) for more details.